sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormTrainingClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.AccountProperties;
+import com.azure.ai.formrecognizer.models.CustomFormModel;
+import com.azure.core.util.Context;
+
+import java.time.Duration;
+
+/**
+ * Code snippet for {@link FormTrainingClient}
+ */
+public class FormTrainingClientJavaDocCodeSnippets {
+    private FormTrainingClient formTrainingClient = new FormRecognizerClientBuilder().buildClient()
+        .getFormTrainingClient();
+
+    /**
+     * Code snippet for {@link FormTrainingClient} initialization
+     */
+    public void formTrainingClientInInitialization() {
```

Comment:
Similar updates as Async client

Reply:
Not sure about this. Can you give more info?

Reply:
I meant to say follow the same comments as given in Async client file for this file changes.